app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class RecipesController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @recipes = current_user.recipes.all
+  end
+
+  def show
```

Comment:
- [ ] Kindly add a modal to choose an inventory in the recipe details page as in the wireframe as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=Should%20have%20a%20modal%20to%20choose%20an%20inventory%20to%20create%20a%20shopping%20list%20with%2C%20as%20in%20the%20wireframe.).

Reply:
- [ ] Just as the previous reviewer mentioned, when I click on generate shopping list, I do not see any modal instead it takes me to a shopping list page that is blank, kindly add the modal